core/src/main/scala/cats/mtl/Handle.scala

```diff
@@ -217,6 +218,42 @@ private[mtl] trait HandleInstances extends HandleLowPriorityInstances {
     }
 }
 
-object Handle extends HandleInstances {
+object Handle extends HandleInstances with HandleCrossCompat {
+
   def apply[F[_], E](implicit ev: Handle[F, E]): Handle[F, E] = ev
+
+  def allowF[F[_], E]: AdHocSyntaxTired[F, E] =
+    new AdHocSyntaxTired[F, E](())
+
+  @scala.annotation.nowarn("msg=dubious usage of method hashCode with unit value")
+  private[mtl] final class AdHocSyntaxTired[F[_], E](private val unit: Unit) extends AnyVal {
```

Comment:
Huh, that's a weird warning 😅 instead of `Unit` we could also use `dummy: Byte` or something else.

Reply:
fixed in https://github.com/typelevel/cats-mtl/pull/619/commits/a70b9f4e5d9f0ca9c50b4f932ca5387aaa7777e9